spec/system/bus_stops/managing_bus_stops_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+describe 'managing stops as an admin' do
+  let(:admin) { create :user, :admin }
+  let!(:bus_stop) { create :bus_stop }
+  before :each do
+    when_current_user_is admin
+    visit manage_bus_stops_url
+  end
+  context 'delete button' do
+    it 'deletes the specific bus stop' do
+      expect(page).to have_selector 'table.manage tbody tr', count: 1
+      within 'tr', text: bus_stop.name do
+        click_button 'Delete'
+      end
+      page.driver.browser.switch_to.alert.accept
+      expect(page).to have_selector 'p.notice',
+        text: "#{bus_stop.name} has been deleted."
+      expect(page).not_to have_selector 'table.manage tbody tr', text: bus_stop.name
+    end
+  end
+  context 'edit button' do
+    it 'redirects to edit bus stop page' do
+      expect(page).to have_selector 'table.manage tbody tr', count: 1
+      within 'tr', text: bus_stop.name do
+        click_link 'Edit'
+      end
+      expect(page).to have_content "Editing #{bus_stop.name}"
+      expect(page.current_url).to end_with edit_bus_stop_path(bus_stop.hastus_id)
+    end
+  end
+end
+
+describe 'viewing outdated' do
+  let(:admin) { create :user, :admin }
+  let!(:present_stop) { create :bus_stop }
+  let!(:date) { Date.today }
+  let!(:old_stop_1) { create :bus_stop, updated_at: (date - 2.months) }
+  let!(:old_stop_2) { create :bus_stop, updated_at: (date - 3.months) }
+  before :each do
```

Comment:
If you're going to have a `before :each` I think you should define your variables in it rather than `let` blocks. I don't think there's any need for both in this case.

Reply:
I disagree. There is no reason to not have both unless she were using a mix of `let` variables _and_ instance variables. The reason there isn't functional, it's decreasing confusion upon reading it. If she had a `@stop` and a `let :stop do create :stop` it would make any future developers pause and ask "wait, why? what's the difference?" And if there is none, that I'd say is a reason for refactor. But, she is not, it just _looks_ bad to you, but some people just would prefer to be consistent and use `let` across the board rather than futz with instance variables. I say "let" them. hahaha

Reply:
Right, you can use `let` blocks to define objects because you prefer how they work to instance variables, and yet still have some actions that you need to perform before each test (logging in, visiting the page, etc.). Nothing wrong with that.